docs/core/run-time-config/compilation.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Compilation config settings
+description: Learn about run-time settings that configure how the JIT compiler works for .NET Core apps.
```

Comment:
Are these settings for the .net core **runtime** or settings for the app at **run-time**?

Reply:
Good question. I'd assumed they were settings for the app at run time, but now that I think about it, they would affect any .NET Core app that's running at the time on the computer. So maybe they are settings for the .NET Core runtime.

Reply:
Thinking about it more, it seems like the environment variables would affect any running .NET Core app, but the runtimeconfig.json file is app-specific. So I believe these are settings for the app at run time.